design/templates/solution_design.md

```diff
@@ -67,5 +70,41 @@
 ## Open questions
 [//]: # (Add any question that is still open. It makes it easier for the reader to have the open questions accumulated here istead of them being acattered along the doc)
 
-## Implementation plan
+## Delivery plan
 [//]: # (Break the solution into tasks)
+
+### Total
+[//]: # (Write the total estimation)
+
+### Parallelism
+[//]: # (Suggest how many team members can work on the solution in parallel)
+
+### Ramp up
+[//]: # (Estimate the tasks, which related of leaning new area [technology, language, feature, etc.])
+
+### Infrastructure and preparations
+[//]: # (Estimate the tasks, which related of implementing any needed infrastructure [jenkins file, scripts, Docker build, etc.])
+
+### Designs 
+[//]: # (Estimate the tasks)
+
+### Spikes and researches
+[//]: # (Estimate the tasks)
+
+### Implementation 
+[//]: # (Estimate the tasks)
+
+### Testing 
+[//]: # (Estimate the tasks)
+
+### Security 
+[//]: # (Estimate the tasks)
+
+### Docs 
+[//]: # (Estimate the tasks)
+
+### Demo 
+[//]: # (Estimate the tasks)
+
+### Left overs and refactoring 
+[//]: # (Estimate the tasks)
```

Comment:
Estimate the tasks that need to be done as part of this effort but will not be delivered in the initial release and will be developed later.